Sources/ArcGISToolkit/Components/FeatureFormView/GroupView.swift

```diff
@@ -84,3 +84,34 @@ struct GroupView<Content>: View where Content: View {
         }
     }
 }
+
+extension GroupView {
+    /// A view displaying a label and description of a `GroupFormElement`.
+    struct Header: View {
+        let element: GroupFormElement
+        
+        var body: some View {
+            VStack(alignment: .leading) {
+                // Text views with empty text still take up some vertical space in
+                // a view, so conditionally check for an empty title and description.
+                if !element.label.isEmpty {
+                    Text(element.label)
+                        .multilineTextAlignment(.leading)
+                        .font(.title2)
+                        .foregroundColor(.primary)
```

Comment:
We'll need to manually add in accessibility identifiers now, we previously got them for free using the other `DisclosureGroup` initializer.

```suggestion
                        .foregroundColor(.primary)
                        .accessibilityIdentifier("\(element.label)")
```

Test case 6.1 and 6.2 will need slight changes as well:

```swift
    func testCase_6_1() {
        let app = XCUIApplication()
        let collapsedGroupFirstElement = app.staticTexts["Single Line Text"]
        let collapsedGroup = app.staticTexts["Group with Multiple Form Elements 2"]
        let expandedGroupFirstElement = app.staticTexts["MultiLine Text"]
        let expandedGroup = app.staticTexts["Group with Multiple Form Elements"]
        let expandedGroupDescription = app.staticTexts["Group with Multiple Form Elements Description"]
        let formTitle = app.staticTexts["group_formelement_UI_not_editable"]
        let formViewTestsButton = app.buttons["Feature Form Tests"]
```

```swift
    func testCase_6_2() {
        let app = XCUIApplication()
        let formTitle = app.staticTexts["group_formelement_UI_not_editable"]
        let formViewTestsButton = app.buttons["Feature Form Tests"]
        let showElementsButton = app.buttons["show invisible form element"]
        let hiddenElementsGroup = app.staticTexts["Group with children that are visible dependent"]
        let hiddenElementsGroupDescription = app.staticTexts["Group with children that are visible dependent Description"]
        let groupElement = app.staticTexts["single line text 3"]
```